### PR TITLE
QMediathekView: hardening

### PR DESCRIPTION
**Pull request by glitsj16**

### CODE REVIEW

File: etc/profile-m-z/QMediathekView.profile [hunk 1]
```diff
@@ -27,29 +27,50 @@ include disable-programs.inc
 include disable-shell.inc
 include disable-xdg.inc
 
+whitelist ${HOME}/.config/QMediathekView
+whitelist ${HOME}/.local/share/QMediathekView
```
Comment (kmk3) [inline etc/profile-m-z/QMediathekView.profile:31]: Why no `mkdir`?

File: etc/profile-m-z/QMediathekView.profile [hunk 2]
```diff
@@ -27,29 +27,50 @@ include disable-programs.inc
...
+whitelist ${HOME}/.config/vlc
+whitelist ${HOME}/.config/xplayer
+whitelist ${HOME}/.local/share/totem
+whitelist ${HOME}/.local/share/xplayer
+whitelist ${HOME}/.mplayer
+
+whitelist ${DOWNLOADS}
+whitelist ${VIDEOS}
+
 whitelist /usr/share/qtchooser
```
Comment (kmk3) [inline etc/profile-m-z/QMediathekView.profile:45]:
```suggestion
whitelist ${DOWNLOADS}
whitelist ${VIDEOS}

whitelist ${HOME}/.config/mpv
whitelist ${HOME}/.config/smplayer
whitelist ${HOME}/.config/totem
whitelist ${HOME}/.config/vlc
whitelist ${HOME}/.config/xplayer
whitelist ${HOME}/.local/share/totem
whitelist ${HOME}/.local/share/xplayer
whitelist ${HOME}/.mplayer
whitelist /usr/share/qtchooser
```

This makes it easier to sort everything and I think it's more similar to other
profiles.

File: etc/profile-m-z/QMediathekView.profile (hunk 1 above)
Comment (glitsj16) [reply etc/profile-m-z/QMediathekView.profile:33]: Fixed!

File: etc/profile-m-z/QMediathekView.profile (hunk 2 above)
Comment (glitsj16) [reply etc/profile-m-z/QMediathekView.profile]: Looks better like that indeed. Thanks!

**Review by kmk3 — APPROVED** [review]

**Comment by netblue30** [comment]
merged, thanks!

**Pull request merged**